CROSS-CUTTING-CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+<!--
```

Comment:
nit: For a title what about `WORKING-WITH-SUBMODULES.md` ?

Reply:
`CONTRIBUTING-SUBMODULES.md` ? 

or just putting the content into the existing `CONTRIBUTING.md` ? 

🤷🏻‍♀️

Reply:
It's so small now, I'm fine w/ just putting it in `CONTRIBUTING.md`

Reply:
will move there